01-Git_Bash_Make/bash_script~

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir guess_a_number
+cp main.c guess_a_number/
+tar -czvf guess_a_number.tar.gz guess_a_number/
+mkdir guess_a_number/release
+cp guess_a_number.tar.gz guess_a_number/release/
```

Comment:
@vometel,
Please don't commit junk. (and this particular should be wiped out from the commit)

As soon as you'r using `.gitignore` and you editor creates backup files, it makes sense to ignore them as well.

But generally it's rather bad approach to add all the files and commit them without review (either before or just after committing).